### PR TITLE
EWB-2448: Pandapower Translator Example

### PR DESCRIPTION
**Pull request by Lax125**
# Description

Adds an example of a node breaker network model (of the 13 node test feeder) being translated into a Pandapower model.

# Associated tasks:

Tasks blocking this one:
- [x] Merge https://github.com/zepben/evolve-sdk-python/pull/123
- [x] Merge https://github.com/zepben/edith-sdk/pull/4
- [x] Merge https://github.com/zepben/pp-translator/pull/20

Tasks this is blocking:
- None

# Checklist:

- [x] I have performed a self review of my own code.
- [x] I have added/updated unit tests for these changes, and if not I have explained why they are not necessary. (No tests needed for example code
- [x] I have updated the changelog.
- [x] I have commented my code in any hard-to-understand or hacky areas.
- [x] I have handled all new warnings generated by the compiler or IDE.

### CODE REVIEW

**Comment by Lax125** [comment]
I tried running a load flow simulation on the test network, and it looks like the `LinearShuntCompensator`s are not supported. They are currently causing disconnections in the resulting busbranch model.

File: src/zepben/examples/ieee_13_node_test_feeder.py [hunk 1]
```diff
@@ -8,108 +8,117 @@
 import numpy
 from zepben.evolve import AcLineSegment, Disconnector, PowerTransformer, TransformerFunctionKind, NetworkService, Terminal, PowerTransformerEnd, EnergyConsumer, \
     PerLengthSequenceImpedance, PhaseCode, EnergyConsumerPhase, SinglePhaseKind, LinearShuntCompensator, ShuntCompensatorInfo, PhaseShuntConnectionKind, Feeder, \
-    LvFeeder
+    LvFeeder, BaseVoltage, EnergySource, Breaker
 
 __all__ = ["network"]
 
 FEET_PER_MILE = 5280
+METRES_PER_FOOT = 0.3048
```
Comment (Lax125) [inline src/zepben/examples/ieee_13_node_test_feeder.py:16]: The pandapower model wouldn't make sense if I kept using feet as the base unit

**Comment by Lax125** [comment]
@kgreav should the example include running a load flow simulation? If so, I'll need to add support for `LinearShuntCompensator`s in the pandapower translator. That or I got the network topology wrong (acls, compensator, and consumer are connected in series).

**Comment by kgreav** [comment]
Sorry didn't get an email notification for your comment last week, seems like because I wasn't reviewer/tagged.

I guess it doesn't really make sense to put it in a PP model and not run a load flow, so yes we should.

I'm not sure if we actually have to add support for LinearShuntCompensators - maybe ask in dev if it makes sense to model them for load flows? And then if it does update the pp-translator to handle them.

**Comment by Lax125** [comment]
Read the description for `ShuntCompensator`, it only should have one terminal. I fixed the network topology. I don't think they add any capacitance to the busbranch model though.

File: src/zepben/examples/ieee_13_node_test_feeder.py [hunk 2]
```diff
@@ -8,125 +8,134 @@
...
-lsc_611 = LinearShuntCompensator(mrid="lsc_611", terminals=[lsc_611_t1, lsc_611_t2], asset_info=lsc_611_info)
+lsc_611 = LinearShuntCompensator(mrid="lsc_611", terminals=[lsc_611_t], asset_info=lsc_611_info)
 
 ###########################
 # SETTING LINE IMPEDANCES #
 ###########################
 def plsi_from_z_per_mile(mrid: str, *impedances: Tuple[float, float]):
-    r_per_foot, x_per_foot = numpy.mean(impedances, axis=0) / FEET_PER_MILE
-    return PerLengthSequenceImpedance(mrid=mrid, r=r_per_foot, x=x_per_foot)
+    r_per_metre, x_per_metre = numpy.mean(impedances, axis=0) / (METRES_PER_FOOT * FEET_PER_MILE)
```
[the remark this answers is not in the document]
Comment (Lax125) [reply src/zepben/examples/ieee_13_node_test_feeder.py:137]: It works just fine

[the remark this answers is not in the document]
Comment (Lax125) [reply src/zepben/examples/ieee_13_node_test_feeder.py:137]: @sonatype-lift ignore

**Comment by Lax125** [comment]
The change to support modelling `LinearShuntCompensator` in the pandapower model is probably going to be in `pp_translators`. That can happen after this is merged.

File: src/zepben/examples/ieee_13_node_test_feeder.py [hunk 3]
```diff
@@ -8,125 +8,134 @@
...
 
 __all__ = ["network"]
 
 FEET_PER_MILE = 5280
+METRES_PER_FOOT = 0.3048
 
+#################
+# BASE VOLTAGES #
+#################
+mv = BaseVoltage(mrid="mv", nominal_voltage=4_160)
```
Comment (kgreav) [inline src/zepben/examples/ieee_13_node_test_feeder.py:21]: what does the underscore between the numbers do here?

**Review by kgreav — APPROVED** [review]

**Pull request merged**